### ISSUE TITLE
Date.setTime() demo doesn't work

### ISSUE THREAD

**Issue by cyrusae** [mdn/content#15019]
### MDN URL

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/setTime

### What specific section or headline is this issue about?

"Try it" console

### What information was incorrect, unhelpful, or incomplete?

The output for me is 
```
> Object {  }
> Object {  }
```

### What did you expect to see?

According to the page, the intended output is
```
console.log(event1);
Thu Jul 01 1999 00:00:00 GMT+0200 (CEST)
Thu Jul 01 1999 00:00:00 GMT+0200 (CEST)
```

### Do you have anything more you want to share?

I'm in PST, if the timezone matters for this.

### MDN metadata

<details>
<summary>Page report details</summary>

* Folder: `en-us/web/javascript/reference/global_objects/date/settime`
* MDN URL: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/setTime
* GitHub URL: https://github.com/mdn/content/blob/main/files/en-us/web/javascript/reference/global_objects/date/settime/index.md
* Last commit: https://github.com/mdn/content/commit/cf607d68522cd35ee7670782d3ee3a361eaef2e4
* Document last modified: 2021-07-20T18:47:57.000Z

</details>

**Comment by sideshowbarker** [comment]
> The output for me is
> 
> ```
> > Object {  }
> > Object {  }
> ```

Yes, for me too. So this seems to be a bug in the interactive example that we really need to fix.

**Comment by NiedziolkaMichal** [comment]
It's related to PR [#710](https://github.com/mdn/bob/pull/745)

**Comment by sideshowbarker** [comment]
> It's related to PR [#710](https://github.com/mdn/bob/pull/745)

ah, right, thanks — so the workaround is to do `console.log(event1.toString())` rather than just `console.log(event1)`
